doc/src/docs/EngineeringReference/13e-EncyclopaedicRefs.md

```diff
@@ -644,6 +644,115 @@ Several things should be noted about the fraction of heat that is radiated direc
+
+* The entity being modeled is essentially a heat exchanger with flowing water on one side and air on the other.
+
+* Since the water is changing temperature but the air is “not”, we effectively have a similar sort of heat exchanger formulation as the low temperature radiant heating system, but here the water temperature varies while the air temperature is “constant”.  Thus, an effectiveness-NTU heat exchanger formulation is best and allows the calculation of total heat gain to the panel and outlet water temperature.
+
+* The total net heat loss to the zone is determined using a user defined coefficient to calculate a zone temperature.  This “fraction” radiant parameter defines what portion of the zone temperature is based on the zone MRT with the remainder determined by the zone MAT.  While this user defined coefficient remains constant throughout the simulation in keeping with the simple nature of this model, it is not the same as defining a fixed radiant/convective split since this will vary with changing MRT and MAT.  More information on this is given below.
+
+* The load met by the system is included in the zone air heat balance as any other system’s impact is.  The impact of the system includes the convective load, the radiant gain that is distributed to people (assumed that it is converted into a convective loss), and the increased convection loss from the zone surfaces.
+
+* The radiant loss/gain is distributed to the surfaces as per the radiant surface list that lists which surfaces the radiant cooling panel will interact with and what fraction of radiation each surface will “see”.  This is similar to the existing radiant-convective baseboard model and these fractions are user inputs.
+
+* The model is an HVAC level model but interacts with the zone heat balances much like the radiant system and radiant/convective baseboard models.
+
+* Controls are based on controlling water flow rate (like the hydronic radiant system) to meet the thermostatic setpoint temperature of the unit with control types potentially including MAT, MRT, OpTemp, and/or ODB/OWB much like the low temperature radiant systems.  Controls based on standard air system thermostatic controls are also possible.
+
+* In the event that condensation is predicted on the surface of a panel, the model uses the same condensation control options as the low temperature radiant system.  These include: ignoring the problem and allowing the simulation to continue, shutting the system down when condensation is predicted, and a variable response that tries to keep the system running at a lower output while avoiding condensation conditions.
+
+* In order to avoid any violations of the Second Law of Thermodynamics or heating by the system, the cooling panel will be shut off if the inlet water temperature is above the zone temperature.
```

Comment:
> In order to avoid any violations of the Second Law of Thermodynamics or heating by the system

I like this.  It's like you are saying:

In order to avoid **AN ALIEN INVASION**, or, you know, a spider